crates/vm/levm/src/gas_cost.rs

```diff
@@ -747,3 +747,19 @@ pub fn fake_exponential(factor: u64, numerator: u64, denominator: u64) -> Result
             .ok_or(InternalError::ArithmeticOperationOverflow)?,
     ))
 }
+
+pub fn max_message_call_gas(current_call_frame: &CallFrame) -> Result<U256, VMError> {
+    let mut potential_remaining_gas = current_call_frame
+        .gas_limit
+        .checked_sub(current_call_frame.gas_used)
+        .ok_or(VMError::OutOfGas(OutOfGasError::MaxGasLimitExceeded))?;
+
+    potential_remaining_gas =
+        potential_remaining_gas
+            .checked_sub(potential_remaining_gas.checked_div(64.into()).ok_or(
```

Comment:
Hmm, couldn't we add a comment explaining why we divide by 64 here?

Lean and Dami explained to me why that is done, but maybe it would be worth while to add a link to that explanation. 
```suggestion
             // This division by 64 comes from the specification:  https://eips.ethereum.org/EIPS/eip-150
            .checked_sub(potential_remaining_gas.checked_div(64.into()).ok_or(
```

Reply:
True, I will update you when I change it.

Reply:
https://github.com/lambdaclass/ethrex/pull/1460/commits/25a12e7cbcb4df75e3b1a2f1c4ba1ebb6a0b9d36
I changed some other things more related to this function but I think that the comment on top of it should suffice.